preview/Win7Msix/Win7MSIXInstaller/InstallUI.hpp

```diff
@@ -28,13 +28,15 @@ class UI
 public:
     HRESULT ShowUI();
     HRESULT LaunchInstalledApp();
-    void ConfirmAppCancel();
+    void ConfirmAppCancel(HWND parentHWnd);
 
     static HRESULT Make(_In_ MsixRequest* msixRequest, _Out_ UI** instance);
     ~UI() {}
 private:
     MsixRequest* m_msixRequest = nullptr;
 
+    HWND hWnd = NULL; //Parent Window Hwnd
+    std::wstring installOrUpdateText = GetStringResource(IDS_STRING_INSTALLTEXT); /// Default button and install screen UI text is 'Install'

```

Comment:
>installOrUpdateText [](start = 17, length = 19)

sorry, i should have mentioned, the convention we're using is to prefix member variables with m_ (see lines immediately below)

Reply:
Resolved.